src/v2/view-builder/views/IdentifierView.js

```diff
@@ -140,7 +153,19 @@ const Body = BaseForm.extend({
 
 export default BaseView.extend({
   Body,
-  Footer: IdentifierFooter,
+
+  initialize() {
+    // We override the default IdentifierFooter by passing in our custom shouldShowForgotPasswordLink() method.
```

Comment:
So we can just say 

`Override Footer by overriding showForgotPasswordLink method`

Reply:
sounds good, done